tests/upgrade/basic/task.yaml

```diff
@@ -54,7 +54,9 @@ execute: |
     # plumbing for that into the snap command.
     if [ -e /sys/kernel/security/apparmor ]; then
         echo Hello > /var/tmp/myevil.txt
-        test-snapd-tools.cat /var/tmp/myevil.txt && exit 1 || true
+        if test-snapd-tools.cat /var/tmp/myevil.txt; then
```

Comment:
Nice, thank you. I find this much easier to read (feels much more common to the go/C/python style I'm using so much more than sh).